doc/userdata.txt

```diff
@@ -52,17 +52,15 @@ finds.  However, certain types of user-data are handled specially.
    This content is "cloud-config" data.  See the examples for a
    commented example of supported config formats.
 
- * Cloud Boothook 
+ * Cloud Boothook
    begins with #cloud-boothook or Content-Type: text/cloud-boothook
 
    This content is "boothook" data.  It is stored in a file under
    /var/lib/cloud and then executed immediately.
 
    This is the earliest "hook" available.  Note, that there is no
    mechanism provided for running only once.  The boothook must take
-   care of this itself.  It is provided with the instance id in the
-   environment variable "INSTANCE_ID".  This could be made use of to
-   provide a 'once-per-instance'
+   care of this itself.
```

Comment:
should a suggestion on how to implement this be left here or is it better just leave this out?

Reply:
I think it's fine as-is. I'm actually thinking we should nuke this file as it's not rendered out to the docs anywhere and says the same thing as `format.rst` but in a worse way.